custom/enikshay/management/commands/get_vouchers_for_bets.py

```diff
@@ -60,10 +60,12 @@ def handle(self, domain, repeater_id, filename, **options):
                     payload = json.loads(record.get_payload())['voucher_details'][0]
 
                     voucher_id = record.payload_id
-                    person_case = get_person_case_from_voucher(domain, voucher_id)
                     voucher_case = accessor.get_case(voucher_id)
-                    payload[u'PersonId'] = person_case.case_id
-                    payload[u'AgencyId'] = voucher_case.get_case_property('voucher_fulfilled_by_id')
+                    # Could also be person_case.get_case_property('person_id')
+                    payload[u'PersonID'] = voucher_case.get_case_property('person_enikshay_id')
```

Comment:
I don't think voucher cases have this property. Where did you find that? 
I think better to use `person_case.get_case_property('person_id')`